MinPlatformPkg/Test/Library/TestPointCheckLib/TraditionalMmTestPointCheckLib.c

```diff
@@ -77,95 +65,31 @@ GLOBAL_REMOVE_IF_UNREFERENCED UINTN                           mGcdIoMapNumberOfD
 
 EFI_MEMORY_ATTRIBUTES_TABLE  *mUefiMemoryAttributesTable;
 
-GLOBAL_REMOVE_IF_UNREFERENCED ADAPTER_INFO_PLATFORM_TEST_POINT_STRUCT  mTestPointStruct = {
-  PLATFORM_TEST_POINT_VERSION,
-  PLATFORM_TEST_POINT_ROLE_PLATFORM_IBV,
-  {TEST_POINT_IMPLEMENTATION_ID_PLATFORM_SMM},
-  TEST_POINT_FEATURE_SIZE,
-  {0}, // FeaturesImplemented
-  {0}, // FeaturesVerified
-  0,
-};
-
-GLOBAL_REMOVE_IF_UNREFERENCED UINT8  mFeatureImplemented[TEST_POINT_FEATURE_SIZE];
-
-/**
-  This service verifies SMRR configuration at the End of DXE.
-
-  Test subject: SMRR.
-  Test overview: Verify SMRR is aligned and SMRR matches SMRAM_INFO.
-  Reporting mechanism: Set ADAPTER_INFO_PLATFORM_TEST_POINT_STRUCT.
-                       Dumps SMRR and SMRAM_INFO.
-
-  @retval EFI_SUCCESS         The test point check was performed successfully.
-  @retval EFI_UNSUPPORTED     The test point check is not supported on this platform.
-**/
-EFI_STATUS
-EFIAPI
-TestPointSmmEndOfDxeSmrrFunctional (
-  VOID
-  )
-{
-  EFI_STATUS  Status;
-  BOOLEAN     Result;
-
-  if ((mFeatureImplemented[TEST_POINT_INDEX_BYTE6_SMM] & TEST_POINT_BYTE6_SMM_END_OF_DXE_SMRR_FUNCTIONAL) == 0) {
-    return EFI_SUCCESS;
-  }
-
-  DEBUG ((DEBUG_INFO, "======== TestPointSmmEndOfDxeSmrrFunctional - Enter\n"));
-
-  Result = TRUE;
-  Status = TestPointCheckSmrr ();
-  if (EFI_ERROR(Status)) {
-    Result = FALSE;
-  }
-
-  if (Result) {
-    TestPointLibSetFeaturesVerified (
-      PLATFORM_TEST_POINT_ROLE_PLATFORM_IBV,
-      NULL,
-      TEST_POINT_INDEX_BYTE6_SMM,
-      TEST_POINT_BYTE6_SMM_END_OF_DXE_SMRR_FUNCTIONAL
-      );
-  }
-
-  DEBUG ((DEBUG_INFO, "======== TestPointSmmEndOfDxe - Exit\n"));
-  return EFI_SUCCESS;
-}
-
 /**
-  This service verifies the validity of the SMM memory atttribute table at SMM Ready To Lock.
-
+  This service verifies the validity of the SMM memory attribute table at SMM Ready To Lock.
   Test subject: SMM memory attribute table.
   Test overview: Verify the SMM memory attribute table is reported.
                  Verify image code/data is consistent with the SMM memory attribute table.
                  Verify the GDT/IDT/PageTable is RO, data is NX, and code is RO.
   Reporting mechanism: Set ADAPTER_INFO_PLATFORM_TEST_POINT_STRUCT.
                        Dumps the SMM memory attribute table and SMM image information.
-
   @retval EFI_SUCCESS         The test point check was performed successfully.
   @retval EFI_UNSUPPORTED     The test point check is not supported on this platform.
```

Comment:
Do you know why soome of these newlines were removed?

It helped separate the paragraphs more clearly before. Can you please go through and any back that are removed now?

Reply:
Fixed.